src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        bulldozer.stopWork();
 
+        Machine truck = new Truck();
```

Comment:
Try to create an array of machines and call required methods. In this case, you will have a complete polymorphism

Reply:
overall LGTM